mne/epochs.py

```diff
@@ -3667,7 +3667,7 @@ def average_movements(epochs, head_pos=None, orig_sfreq=None, picks=None,
 
 
 @verbose
-def make_fixed_length_epochs(raw, duration=1., preload=False,
+def make_fixed_length_epochs(raw, duration=1., overlap=0., preload=False,
```

Comment:
@sfc-neuro you might break the code of people by inserting a new param in the middle. We usually but it just before verbose.

Reply:
Thanks, moved.